### PR TITLE
[Bug] French formatting on poster page

### LINKED ISSUE #10071: 🐛 Incorrect French formatting in the skills requirement section
## 🐛 Bug

There is incorrect formatting in the skills requirement section in French.

## 🦋 Expected Behaviour

Text should be formatted correctly in French.

## 📋 Steps to Reproduce

1. Navigate to `/fr/browse/pools/:pool-id#skill-requirements`
2. Observe incorrect formatting in first two paragraphs

## 📸 Screenshots

Screen Shot 2024-04-15 at 17 01 19
Screen Shot 2024-04-15 at 16 56 32

## 🙋‍♀️ Proposed Solution

- Add space between the word _compétences_ and _Facultatives_
- Remove comma after the word _Facultatives_
- Replace double quotes with _guillemets_

> [!NOTE]  
> I would suggest that there should be no double quotes in English for the words Required and Optional. They are already a different colour and emphasized with `<strong>`.

## ✅ Acceptance Criteria

A set of assumptions which, when tested, verify that the bug was addressed.

- [x] space exists between the word _compétences_ and _Facultatives_
- [x] no comma exists after the word _Facultatives_
- [x] no double quotes are rendered in French

### PR DESCRIPTION
**Pull request by vd1992**
🤖 Resolves #10071 

## 👋 Introduction

Resolve the typos present, as well as implement the suggestion. I think it looks nicer that way as it cleanly matches the accordion. 

## 🧪 Testing

1. Navigate to `/en/browse/pools/:id`
2. Then in French

## 📸 Screenshot

image

image

### CODE REVIEW

File: apps/web/src/pages/Pools/PoolAdvertisementPage/PoolAdvertisementPage.tsx [hunk 1]
```diff
@@ -758,8 +758,8 @@ export const PoolPoster = ({
               <Text>
                 {intl.formatMessage({
                   defaultMessage:
-                    'In order to apply for this job you will need to be able to demonstrate that you have <strong>all the skills</strong> marked <heavyPrimary>"Required"</heavyPrimary>. If you have any of the <heavySecondary>"Optional"</heavySecondary> skills you are encouraged to include them because it will increase your chances of a job placement.',
-                  id: "eihm6A",
+                    "In order to apply for this job you will need to be able to demonstrate that you have <strong>all the skills</strong> marked <heavyPrimary>Required</heavyPrimary>. If you have any of the <heavySecondary>Optional</heavySecondary> skills you are encouraged to include them because it will increase your chances of a job placement.",
```
Comment (mnigh) [inline apps/web/src/pages/Pools/PoolAdvertisementPage/PoolAdvertisementPage.tsx:761]: Although I think it makes sense to remove the double quotes around **Required** and **Optional**, my suggestion is not part of the issue's acceptance criteria. You'll need to check with the designer first if this change is approved as this is not in the [design file](https://www.figma.com/file/CdYmWlP6SpT17bw7Q9411Z/Candidates---Job-advertisement?type=design&node-id=439-10852&mode=design&t=i6mUwX1Sa8Dhd0Ei-0) (I don't know whether this is the most up to date file, but it is what I could find).

**Review by mnigh — COMMENTED** [review]
Just the one thing to check if you want to remove the double quotes, otherwise looks good.

File: apps/web/src/pages/Pools/PoolAdvertisementPage/PoolAdvertisementPage.tsx (hunk 1 above)
Comment (vd1992) [reply apps/web/src/pages/Pools/PoolAdvertisementPage/PoolAdvertisementPage.tsx]: ⏪ 
https://github.com/GCTC-NTGC/gc-digital-talent/pull/10077/commits/14fd3a9d47e2ec378c1695c79c554881731a0815

**Review by mnigh — APPROVED** [review]
The reduction in ambition is approved 😉.

**Pull request merged**